handsontable/src/dataMap/dataMap.js

```diff
@@ -781,7 +781,7 @@ class DataMap {
     if (this.hot.hasHook('modifyData')) {
       const valueHolder = createObjectPropListener(value);
 
-      this.hot.runHooks('modifyData', physicalRow, this.propToCol(prop), valueHolder, 'get');
+      this.hot.runHooks('modifyData', row, this.propToCol(prop), valueHolder, 'get');
```

Comment:
I am surprised that no test failed. Generally, this is a braking change, a change from physical to visual index.

Reply:
I reverted this change